src/EmbeddedExplorer.ts

```diff
@@ -43,24 +48,30 @@ export class EmbeddedExplorer {
   options: EmbeddableExplorerOptions;
   handleRequest: HandleRequest;
   embeddedExplorerURL: string;
+  embeddedExplorerIFrameElement: HTMLIFrameElement;
+  uniqueEmbedInstanceId: string;
   private disposable: { dispose: () => void };
   constructor(options: EmbeddableExplorerOptions) {
     this.options = options;
     this.validateOptions();
     this.handleRequest = this.options.handleRequest ?? fetch;
+    this.uniqueEmbedInstanceId = uuid();
```

Comment:
since we only need this to be locally unique, would an incrementing counter work as well here?  

could either scope the counter to be outside the class, or stored as a static property and increment in the constructor

could also consider [nanoid](https://www.npmjs.com/package/nanoid) which is just 871 bytes (uuid is a bit heavy for what we need from it at 8kb)